test/v4/orderbookReserve.js

```diff
@@ -479,62 +476,61 @@ contract('OrderbookReserve', async (accounts) => {
             await reserve.getTokenToEthAddOrderHint(tweiAmount, weiAmount);
 
             try {
-                await reserve.getTokenToEthAddOrderHint(tweiAmount, weiAmount.sub(1));
+                await reserve.getTokenToEthAddOrderHint(tweiAmount, weiAmount.sub(new BN(1)));
                 assert(false, "throw was expected in line above.")
             } catch(e) {
                 assert(Helper.isRevertErrorMessage(e), "expected throw but got: " + e);
             }
         })
 
         it("verify get update order hint reverts if Eth value is below min order value", async() => {
-            let tokenWeiDepositAmount = 30 * 10 ** 18;
-            let kncTweiDepositAmount = 600 * 10 ** 18;
-            let ethWeiDepositAmount = 3 * 10 ** 18;
+            let tokenWeiDepositAmount = PRECISION.mul(new BN(30));
```

Comment:
this isn't precision.

its confusing since our precision is same value as token decimals for many tokens.
so this should be one tokne.mul.(x)